examples/connect/streamlit/requirements.txt

```diff
@@ -1,56 +1,4 @@
-altair==5.2.0
-attrs==23.2.0
-blinker==1.7.0
-cachetools==5.3.2
-certifi==2024.2.2
-charset-normalizer==3.3.2
-click==8.1.7
+databricks-sql-connector==3.0.1
```

Comment:
This is more a question than an issue or suggestion: what happened with all of these dependencies? Were they old copy 🍝  or something?

Reply:
I did a pip freeze initially on this example but I like the approach that @plascaray used in his examples better where we only capture the dependencies that we really need, and then let pip handle transitive deps. No need for us to record them here IMO so I was cleaning this up so that it's consistent with the other examples

Reply:
💯 Absolutely 

Thanks!